### PR TITLE
add current ticket price to registration/stats

### PR DESCRIPTION
**Pull request by migetman9**

### CODE REVIEW

File: uber/site_sections/registration.py [hunk 1]
```diff
@@ -923,7 +923,7 @@ def stats(self):
         return json.dumps({
             'badges_sold': c.BADGES_SOLD,
             'remaining_badges': c.REMAINING_BADGES,
-
+            'ticket_price': c.BADGE_PRICE,
```
Comment (kitsuta) [inline uber/site_sections/registration.py]: We named everything else 'badges' in these dumps, is there a particular reason to say "ticket" here?

**Comment by migetman9** [comment]
nope. I'll change it to badges

**Comment by kitsuta** [comment]
+1

File: uber/site_sections/registration.py [hunk 2]
```diff
@@ -923,7 +923,7 @@ def stats(self):
         return json.dumps({
             'badges_sold': c.BADGES_SOLD,
             'remaining_badges': c.REMAINING_BADGES,
-
+            'badges_price': c.BADGE_PRICE,
```
Comment (EliAndrewC) [inline uber/site_sections/registration.py]: ``badge_price`` would probably be better than ``badges_price`` but it's not a big deal either way.

**Pull request merged**